full-v2.yml

```diff
@@ -0,0 +1,2289 @@
+        - "12"
+        - "PreKindergarten"
+        - "Kindergarten"
+        - "PostGraduate"
+        - "Other"
+      hispanic_ethnicity:
+        type: string
+        x-nullable: true
+        x-validation: true
+        enum:
+        - "Y"
+        - "N"
+        - ""
+      home_language:
+        type: string
+        x-nullable: true
+        x-validation: true
+        enum:
+        - English
+        - Albanian
+        - Amharic
+        - Arabic
+        - Bengali
+        - Bosnian
+        - Burmese
+        - Cantonese
+        - Chinese
+        - Dutch
+        - Farsi
+        - French
+        - German
+        - Hebrew
+        - Hindi
+        - Hmong
+        - Ilocano
+        - Japanese
+        - Javanese
+        - Karen
+        - Khmer
+        - Korean
+        - Laotian
+        - Latvian
+        - Malay
+        - Mandarin
+        - Nepali
+        - Oromo
+        - Polish
+        - Portuguese
+        - Punjabi
+        - Romanian
+        - Russian
+        - Samoan
+        - Serbian
+        - Somali
+        - Spanish
+        - Swahili
+        - Tagalog
+        - Tamil
+        - Telegu
+        - Thai
+        - Tigrinya
+        - Turkish
+        - Ukrainian
+        - Urdu
+        - Vietnamese
+      iep_status:
+        type: string
+        x-nullable: true
+      last_modified:
+        type: string
+        format: datetime
+        x-validation: true
+      location:
+        $ref: "#/definitions/Location"
+        x-nullable: true
+      name:
+        $ref: "#/definitions/Name"
+      race:
+        type: string
+        x-nullable: true
+        x-validation: true
+        enum:
+        - "Caucasian"
+        - "Asian"
+        - "Black or African American"
+        - "American Indian"
+        - "Hawaiian or Other Pacific Islander"
+        - "Two or More Races"
+        - "Unknown"
+        - ""
+      school:
+        type: string
+        x-validation: true
+      schools:
+        type: array
+        x-validation: true
+        items:
+          type: string
+      sis_id:
+        type: string
+      state_id:
+        type: string
+        x-nullable: true
+      student_number:
+        type: string
+        x-nullable: true
+      unweighted_gpa:
+        type: string
+        x-nullable: true
+      weighted_gpa:
+        type: string
+        x-nullable: true
+
+  StudentContact:
+    type: object
+    properties:
+      id:
+        type: string
+        x-validation: true
+      district:
+        type: string
+        x-validation: true
+      email:
+        type: string
+        x-nullable: true
+      name:
+        type: string
+      phone:
+        type: string
+        x-nullable: true
+      phone_type:
+        type: string
+        x-nullable: true
+      relationship:
+        type: string
+        x-nullable: true
+      sis_id:
+        type: string
+        x-nullable: true
+      student:
+        type: string
+        x-validation: true
+      type:
```

Comment:
Should we make the type and relationship enums? [values mappings: https://github.com/Clever/pipeline-data-quality/blob/master/valuemapper/contacttypes.yml and https://github.com/Clever/pipeline-data-quality/blob/master/valuemapper/contactrelationships.yml]

Possibly also phone_type.